app/Controllers/Talk.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Controllers\BaseController;
+use App\Models\TalkModel;
+use App\Models\EventModel;
+
+class Talk extends BaseController
+{
+    public function getICS($year) {
+        // Start output buffering
+        ob_start();
+
+        // Get event id by year
+        $eventModel = new EventModel();
+        $event      = $eventModel->getByYear($year);
```

Comment:
I think we need an error check here, because maybe there's no event for the given year.